rdrf/rdrf/helpers/utils.py

```diff
@@ -705,6 +705,31 @@ def get_supported_languages():
     return [Language(pair[0], pair[1]) for pair in settings.LANGUAGES]
 
 
+def get_all_language_codes():
+    from collections import namedtuple
+    from django.conf import settings
+    from langcodes import Language, standardize_tag, LANGUAGE_ALPHA3
+
+    languages = []
+    language_info = namedtuple('Language', ['code', 'name'])
```

Comment:
`namedtuple` creates a new class, so it can be defined once in the file rather than each time `get_all_language_codes` is called.